src/core/tooltip/tooltip.js

```diff
@@ -1,5 +1,6 @@
 /* global window */
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
It's declared in the `.babelrc` (the "pragma" property). We don't have the benefits of the usual notation `/** @jsx createElement */` because we're using Airbnb base rules, which doesn't include the JSX ones.